template/conftest.py

```diff
@@ -1,156 +1,15 @@
-import pytest
-from typing import Optional
-from pathlib import Path
-from ragger.firmware import Firmware
-from ragger.backend import SpeculosBackend, LedgerCommBackend, LedgerWalletBackend
-from ragger.navigator import NanoNavigator, StaxNavigator
-from ragger.utils import app_path_from_app_name
+from ragger.conftest import configuration
```

Comment:
Intentional

Reply:
It's to showcase how to import and use the conf
If the user tries to define a new conf class and modify it's values it won't work for instance
Yes it might pop up in every repo that uses the template but they can very well remove this line in their code